### ISSUE TITLE
support SSD1305 128x64

### ISSUE THREAD

**Issue by xaviergh** [olikraus/u8g2#521]
Hello, I have this [display](https://www.adafruit.com/product/2719) from Adafruit that uses the SSD1305 chip, I have tested successfully using the 128x32 constructor but I can't use the whole display. Is there a way to add that extra space in code?

Thanks.

**Comment by olikraus** [comment]
It is not yet implemented. I can add this. Are you willing to do some testing here?

**Comment by xaviergh** [comment]
That would be great! I have the hardware ready to test, just let me know.

Thank you.

**Comment by olikraus** [comment]
I have created beta release 2.22.1 for further testing. Download from here: https://github.com/olikraus/U8g2_Arduino
The zip file can be installed via the Add zip library manu of the Arduino IDE.

Especially let me know whether the  u8x8/flipmode example works.

**Comment by xaviergh** [comment]
Hello, I did some testing and it seems it still take half the screen. Attached is the result of the flip example:
![u8g2_adafruit_flip](https://user-images.githubusercontent.com/22330184/36754810-e30f270c-1be8-11e8-8445-f387c55546b7.gif)

I ran a hello world to test unicode support with the following:

```
U8G2_SSD1305_128X64_ADAFRUIT_F_4W_HW_SPI u8g2(U8G2_R2, pin_oled_cs, pin_oled_dc, 
pin_oled_reset);
```
Setup:
```
u8g2.begin();  
u8g2.enableUTF8Print();
```
Loop:
```
u8g2.setFont(u8g2_font_ncenB08_tf);
u8g2.clearBuffer();
u8g2.setCursor(0, 15);
u8g2.print(" Hello World 1");
u8g2.setCursor(0, 40);
u8g2.print("Hello World 2");
u8g2.sendBuffer();
delay(1000);
```
Just to let you know I'm using a SAMD chip (Adafruit Feather M0) for the test.

![u8g2_adafruit_hello_world](https://user-images.githubusercontent.com/22330184/36754126-ecd6ff50-1be6-11e8-9954-103eff5508e4.jpg)

**Comment by olikraus** [comment]
ok, thanks for the feedback, I have created another beta:
eRlease 2.22.2 . Again, download from here: https://github.com/olikraus/U8g2_Arduino

**Comment by xaviergh** [comment]
I tested again and it seems to work now, just a small issue, in the picture you can see that the edge is cropped. To fix it I have to add an extra space at the beginning.
![u8g2_adafruit](https://user-images.githubusercontent.com/22330184/36803804-71a8c8ac-1c97-11e8-8bc6-02dcc6b64a08.jpg)
Like this:
`u8g2.print(" Hello World 1");`
Instead of
`u8g2.print("Hello World 1");`

Thank you for your work!

**Comment by olikraus** [comment]
strange... It is not cropped, but the text starts on the upper right and wraps over to the left side.
But initially the u8x8 flipmode example did look better. Does this work with u8x8? Do you see similar effects with u8x8? Can you run the u8x8 graphcs test or other examples?

For the example above, can you post the complete source code?

**Comment by xaviergh** [comment]
I used the u8x8 and it has the same issue.
![u8x8_adafruit](https://user-images.githubusercontent.com/22330184/36806604-4a2093de-1c9f-11e8-8c49-1414dbfe74d2.jpg)
The source code is the PrintUTF8 example from full buffer except that I change the chinese text with "Hello World 2".
However something weird happened, by mistake I used the SSD1306 constructor and it works without any issue (Both u8x8 and u8g2), even tho I know that SS1305 and SS1306 are different chips.
